src/Components/Server/test/Circuits/RevalidatingServerAuthenticationStateProvider.cs

```diff
@@ -185,17 +185,20 @@ public async Task SuppliesCancellationTokenThatSignalsWhenRevalidationLoopIsBein
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19940")]
         public async Task IfValidateAuthenticationStateAsyncReturnsUnrelatedCancelledTask_TreatAsFailure()
```

Comment:
This is super unrelated to this change, but could we rename this file to `RevalidatingServerAuthenticationStateProviderTest.cs`?

Reply:
Addressed this and the issues below. I'll push a commit once @SteveSandersonMS has put his feedback.